### PR TITLE
Update to multiarch 19.08 runtime

### PR DESCRIPTION
**Pull request by Pixelnarium**
Please have a look at my attempt to make it work under the 19.08 runtime.

fixes https://github.com/flathub/net.pcsx2.PCSX2/issues/18

### CODE REVIEW

File: net.pcsx2.PCSX2.json [hunk 1]
```diff
@@ -156,7 +209,7 @@
               {
                 "type": "git",
                 "url": "https://github.com/PCSX2/pcsx2.git",
-                "commit": "74e6c437f2cb1e3303fe71237f0ccb257853d804"
+                "commit": "79db01d7ed01a4f3de06a8d8dc5c5e7099ef9d2e"
```
Comment (LeandroStanger) [inline net.pcsx2.PCSX2.json:212]: Update to v1.6.0-rc
```suggestion
                "commit": "876c3bc72016fc482700b39c3b3ab19404e7d679"
```

**Comment by LeandroStanger** [comment]
[Add icon 128x128](https://discourse.flathub.org/t/a-128x128-icon-is-required-now/386?u=leandrostanger)

**Comment by Pixelnarium** [comment]
Well... since the Icon is from upstream. Simply upscaling it or add skip-icon-check? Pretty stupid restriction if you ask me.

**Comment by LeandroStanger** [comment]
**CC:** @TingPing @bilelmoussaoui

**Comment by bilelmoussaoui** [comment]
> Well... since the icon is from upstream. Simply upscaling it or add skip-icon-check? Pretty stupid restriction if you ask me.

No it's not, we use the 128px version on the store, you can upscale/downscale the current version if there are no other sizes available in upstream tarballs.

**Comment by Pixelnarium** [comment]
Well, I honestly don't know why flatpak can't upscale the icon automatically instead of adding maintenance burden but whatever...

**Pull request merged**